README.md

```diff
@@ -166,62 +124,31 @@ Determines which API the Utility Connect front-end points to
 - `sandbox`: This references our sandbox API. Use this in your development, staging or test environments.
 - `production`: This references our production API. **Only use this in your production app.**
 
-### `config.client`
-
-The name used to reference the client organization.
-
 ### `config.callbacks`
 
 Callback functions triggered at key points in the Utility Connect flow. Expects an object with key/value pairs where keys are as documented below.
 
-**`onClose`**: callback function that is triggered when the Utility Connect component is closed.
-
-**`onError`**: callback function that is triggered when an error occurs during the Utility Connect flow.
-
+**`onCredentialsSubmitted`**: callback function that is triggered when the user has clicked the button to submit their credentials. Passes along a utility_credential_id which can optionally be used by the containing application to poll the status of credential verification via the client's backend (which has awareness of the associated client_user_id).
```

Comment:
Should surround utility_credential_id with back ticks (`utility_credential_id`)

Reply:
Should we provide more explicit guidance on the format of the params for the callback functions? For example, we previously displayed:
```
onCredentialsSubmitted: ({ utility_credential_id }) => { ... }),
```
for this endpoint rather than just the callback name.

I don't know what's more common/helpful for documentation

Reply:
`onCredentialsSubmitted: ({ utility_credential_id }) => { ... }),` is below this